src/data/WeekChefRestauData.ts

```diff
@@ -0,0 +1,20 @@
+import { Cards } from "../models/Types";
+import { onzaRestau ,KitchenMarket,Mashya} from "../assets/homePageImg";
+
+const WeekChefRestauData: Cards = {
+  cards: [
+    {
+      title: "Onza",
+      image: onzaRestau,
+    }, {
+      title: "Kitchen Market",
+      image: KitchenMarket,
```

Comment:
Ensure consistency in naming conventions.

Reply:
I've included updates for the "Restaurants" page in this pull request. However, as these changes are still in progress, please focus your review on the other modifications. I've opened a separate branch named "Restaurants" specifically for refining these changes, which I'll address in a dedicated pull request once finalized.